App/Logic/DataUploadLogic.swift

```diff
@@ -362,24 +362,24 @@ extension Logic.DataUpload {
       do {
         // Send the request
         let requestSize = state.configuration.ingestionRequestTargetSize
-        try await(context.dependencies.networkManager.validateCUN(self.code, lastHisNumber: self.lastHisNumber, symptomsStartedOn: self.symptomsStartedOn, requestSize: requestSize))
+        try Hydra.await(context.dependencies.networkManager.validateCUN(self.code, lastHisNumber: self.lastHisNumber, symptomsStartedOn: self.symptomsStartedOn, requestSize: requestSize))
         } catch NetworkManager.Error.unauthorizedOTP {
           // User is not authorized. Bubble up the error to the calling ViewController
-          try await(context.dispatch(Logic.Loading.Hide()))
+          try Hydra.await(context.dispatch(Logic.Loading.Hide()))
             try context.awaitDispatch(ShowCunErrorAlert(title: L10n.UploadData.VpnError.title, message: L10n.UploadData.ErrorCun.message))
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/DataUploadLogic.swift;
  Line: 369;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 138 characters (line_length)